### PR TITLE
TestCase name format should be all in upper cases.

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
TC names should be in all upper cases. Fixed for recent added test cases. 
Verified in Jenkins menu refresh, http://penguinator.westus2.cloudapp.azure.com/view/Microsoft/job/Microsoft-Refresh-Test-Selection-Menus/40/console

### CODE REVIEW

File: XML/TestCases/FunctionalTests.xml [hunk 1]
```diff
@@ -59,7 +59,7 @@
         <Tags>kvp</Tags>
     </test>
     <test>
-        <testName>KVP-ModifyKeyValues</testName>
+        <testName>KVP-MODIFYKEYVALUES</testName>
```
Comment (lizzha) [inline XML/TestCases/FunctionalTests.xml]: suggest to add dashes to separate words, KVP-MODIFY-KEY-VALUES, which makes it more readable

Comment (lizzha) [reply XML/TestCases/FunctionalTests.xml]: apply to other cases

Comment (juhlee-microsoft) [reply XML/TestCases/FunctionalTests.xml]: Fixed in c27ae1813b48017fdf77bccd9f2689caf43aaba4 & verified in http://penguinator.westus2.cloudapp.azure.com/view/Microsoft/job/Microsoft-Refresh-Test-Selection-Menus/42/console

Comment (juhlee-microsoft) [reply XML/TestCases/FunctionalTests.xml]: Added three people who made them in history. The notification will send out.

**Pull request merged**